exp_build/experiment.js

```diff
@@ -246,6 +269,17 @@ class Experiment {
     };
     this.descriptor.units.push(contributors);
   }
+
+  includeCodeEditor(position = 6) {
```

Comment:
Position is 5 in the default experiment descriptor. Why is it defaulted to 6 here? It should be picked from the default experiment descriptor.